pennylane/tape/tape.py

```diff
@@ -45,6 +41,45 @@
 )
 
 
+OPENQASM_GATES = {
+    "CNOT": "cx",
+    "CZ": "cz",
+    "U3": "u3",
+    "U2": "u2",
+    "U1": "u1",
+    "Identity": "id",
+    "PauliX": "x",
+    "PauliY": "y",
+    "PauliZ": "z",
+    "Hadamard": "h",
+    "S": "s",
+    "S.inv": "sdg",
+    "T": "t",
+    "T.inv": "tdg",
+    "RX": "rx",
+    "RY": "ry",
+    "RZ": "rz",
+    "CRX": "crx",
+    "CRY": "cry",
+    "CRZ": "crz",
+    "SWAP": "swap",
+    "Toffoli": "ccx",
+    "CSWAP": "cswap",
+    "PhaseShift": "u1",
+}
+"""
+dict[str, str]: Maps PennyLane gate names to equivalent QASM gate names.
+
+Note that QASM has two native gates:
+
+- ``U`` (equivalent to :class:`~.U3`)
+- ``CX`` (equivalent to :class:`~.CNOT`)
+
+All other gates are defined in the file qelib1.inc:
+https://github.com/Qiskit/openqasm/blob/master/examples/generic/qelib1.inc
```

Comment:
This link no longer works :/

Reply:
looks like they renamed it 🤔 
```suggestion
https://github.com/Qiskit/openqasm/blob/master/examples/stdgates.inc
```